src/widgets/radio.jsx

```diff
@@ -622,35 +706,32 @@ var choiceTransform = (editorProps, problemNum) => {
     };
 
     var addNoneOfAbove = function(array) {
-        // Pick a random choice to replace with 'None of the above'
-        if (!editorProps.randomize && editorProps.noneOfTheAbove) {
-            // Seed RNG with problemNum
-            var rand = seededRNG(problemNum)();
-            var randomIndex = Math.floor(rand * array.length);
-            var itemToBeReplaced = array[randomIndex];
-
-            // Shift array left so that 'None of the above' is last
-            array.splice(randomIndex, 1);
-            array.push(itemToBeReplaced);
+        var noneOfTheAbove = null;
+
+        array = _.reject(array, function(choice, index) {
+            return choice.isNoneOfTheAbove && ( noneOfTheAbove = choice);
```

Comment:
I liked your find way better. Assignment as an expression is tricky to think about when reading code.